charts/example-configurations/minimal-values.tmpl

```diff
@@ -0,0 +1,277 @@
+#
+# Basic configuration file for Renku 0.7.1
+#
+
+---
+domain_configurations:
+  mainURL:                   &mainURL                       [domain]
+  baseURL:                   &baseURL                       https://[domain]
+  jupyterhubURL:             &jupyterhubURL                 https://[domain]/jupyterhub
+  gatewayURL:                &gatewayURL                    https://[domain]/api
+  jupyterhubAuthCallbackUrl: &jupyterhub_gitlab_callbackUrl https://[domain]/jupyterhub/hub/oauth_callback
+  oauth_redirect_uri:        &oauth_redirect_uri            https://[domain]/api/auth/jupyterhub/token
+  ingressTLS:                &ingressTLS                    [ingress-tls]
+
+  gitlabURL:                 &gitlabURL                     https://[gitlab-domain]
+  gitlabPrefix:              &gitlabPrefix                  [gitlab-prefix]           ## /gitlab if part of Renku, / otherwise
+  gitlabClientId:            &gitlabClientId                [gitlab-application-clientID]
```

Comment:
This is fixed in the newest version. I did not hard-code 'jupyterhub' because the script and template should also work for external GitLab deployments.